aws/session/testdata/credential_source_config

```diff
@@ -52,3 +52,8 @@ sso_account_id = 012345678901
 sso_region = us-west-2
 sso_role_name = TestRole
 sso_start_url = https://THIS_SHOULD_NOT_BE_IN_TESTDATA_CACHE/start
+
+[profile sso_invalid]
+sso_account_id = 012345678901
+sso_role_name = TestRole
```

Comment:
Is there a test where the presence of this profile with "invalid sso" won't throw an error?